packages/cli/src/commands/package-dynamic-plugins/command.ts

```diff
@@ -56,20 +58,42 @@ export async function command(opts: OptionValues): Promise<void> {
       ];
   // run export-dynamic on each plugin package
   for (const pluginPkg of packages) {
-    const { packageDirectory, packageFilePath } = pluginPkg;
+    const { packageDirectory, packageFilePath, packageJson } = pluginPkg;
     if (
       !fs.existsSync(path.join(packageDirectory, 'dist-dynamic')) ||
       forceExport
     ) {
-      Task.log(
-        `Running export-dynamic-plugin on plugin package ${packageFilePath}`,
-      );
-      try {
-        await Task.forCommand(`yarn export-dynamic`, { cwd: packageDirectory });
-      } catch (err) {
+      if (
+        Object.keys(packageJson.scripts as { [key: string]: string }).find(
+          script => script === 'export-dynamic',
+        )
+      ) {
+        Task.log(
+          `Running existing export-dynamic script on plugin package ${packageFilePath}`,
+        );
+        try {
+          await Task.forCommand(`yarn export-dynamic`, {
+            cwd: packageDirectory,
+          });
+        } catch (err) {
+          Task.log(
+            `Encountered an error running 'yarn export-dynamic' on plugin package ${packageFilePath}, this plugin will not be packaged.  The error was ${err}`,
+          );
+        }
+      } else {
         Task.log(
-          `Encountered an error running 'yarn export-dynamic on plugin package ${packageFilePath}, this plugin will not be packaged.  The error was ${err}`,
+          `Using npx @janus-idp/cli to export plugin package ${packageFilePath}`,
```

Comment:
Do we really need to use `npx` here ?
Can't we automatically find the place of the janus-cli being called, and use it to call the `export-dynamic-plugin` ?

Reply:
Yeah, that would probably be better anyways, I'll give it a go.  Thanks!